README.md

```diff
@@ -280,6 +280,10 @@ You can use the following workarounds instead of the **Add** button to help you
 
 # Release Notes
 
+## Version 1.1.30 (January 27, 2023)
+
+- Removed CGI scripts
```

Comment:
Technically this plugin did not have any CGI scripts. What we really did is removal dependency on cgi scripts on SDA. We removed CGI scripts from SDA (counterpart of this plugin) and adjusted this plugin (native-jenkins) to work with updated SDA.

Reply:
@justnoxx, 
Release note entry was updated.